addons/account/models/account_fiscalyear.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+from odoo.exceptions import ValidationError
+from odoo import api, fields, models, _
+
+
+class AccountFiscalYear(models.Model):
+    _name = 'account.fiscalyear'
```

Comment:
technically the word fiscalyear is fiscal year should not it be called:

"account.fiscal.year" or simply "account.year" ?

BTW kudos for the feature.

Reply:
+1
Especially since there exist concepts like fiscal weeks, fiscal months, and fiscal quarters

Reply:
Agreed. Account.fiscal.year looks much better, it are three words after all. +1

Reply:
wtf? dudes, don't you have more important things to do ?? ^^

https://www.odoo.com/survey/start/fiscalyear-or-fiscal-year-157

Reply:
jajaaj, Dude, Since v5 my eyes tick with fiscalyear, now that it is comming alive as a zombie, then I think that should awesome have it correctly named.

When somebody new come and you say:

Open the fical year object then the **logic** thing is try: account.fiscal.year.

Maintain the other is as you say "nostalgic".

;-)

nice the survey

Reply:
@qdp-odoo It is soooo important! I took the liberty to tweet for even more votes in your survey! https://twitter.com/Yenthe666/status/932514613266206720 😉 😆

Reply:
ha ha dudes, you are really crazy ><

Reply:
FYI:

Answer Choices | User Responses
-- | --
account.fiscalyear, I'm nostalgic of the v8. | 21.62 % 8
account.fiscal.year, I do not compromise with etymology of technical object's technical names. | 48.65 % 18
account.year, size don't matter. | 2.7 % 1
account.i.do.not.c-year, seriously, don't you have anything more important to do? | 27.03 % 10

So let's go for the majority **account.fiscal.year**

Reply:
Hooray! This will feel like a victory every damn time I see or use it.